extensions/amp-a4a/0.1/a4a-variable-source.js

```diff
@@ -56,7 +56,6 @@ const WHITELISTED_VARIABLES = [
   'SOURCE_URL',
   'TIMESTAMP',
   'TIMEZONE',
-  'TIMEZONE_CODE',
```

Comment:
We should definitely remove the variable from the allowed list so it won't get initialized below. However the list also serves as a documentation on what is supported in the a4a environment. Maybe we should host such a document list somewhere else, or add a comment here. 
Not blocking.